metrics-collectd/src/main/java/com/codahale/metrics/collectd/Sanitize.java

```diff
@@ -0,0 +1,43 @@
+package com.codahale.metrics.collectd;
+
+/**
+ * @see <a href="https://collectd.org/wiki/index.php/Naming_schema>Collectd naming schema</a>
+ */
+final class Sanitize {
+
+    private static final int MAX_LENGTH = 63;
+
+    private static final char DASH = '-';
+    private static final char SLASH = '/';
+    private static final char NULL = '\0';
+    private static final char UNDERSCORE = '_';
+
+    static String name(String name) {
+        return sanitize(name, DASH, SLASH, NULL);
+    }
+
+    static String instanceName(String instanceName) {
+        return sanitize(instanceName, SLASH, NULL);
+    }
+
+    private static String sanitize(String string, char... reservedChars) {
```

Comment:
Instead of using varags, I think it's more reasonable to declare the reserverd chars as constants:
```java
private static final List<Character> INSTANCE_RESERVED = Arrays.asList(SLASH, NULL);
private static final List<Character> NAME_RESERVED = Arrays.asList(DASH, SLASH, NULL);
```
As an advantage, we can reuse the `contains` method of `List` instead wring an own over a char array.